x/dsmr/node.go

```diff
@@ -453,3 +453,10 @@ func (p pChain) GetValidatorSet(context.Context, uint64, ids.ID) (map[ids.NodeID
 
 	return result, nil
 }
+
+func (pChain) GetCurrentValidatorSet(
+	context.Context,
+	ids.ID,
+) (map[ids.ID]*snowValidators.GetCurrentValidatorOutput, uint64, error) {
+	return nil, 0, nil
+}
```

Comment:
why does this return empty?

Reply:
The method is currently not being used. I'll add an error here to indicate that.
( it will be removed in my subsequent PR ).